### PR TITLE
add leading position for tags

### PR DESCRIPTION
**Pull request by mwonng**
## Change summary
Add a position ahead of node type name with tooltip as pictures.

## What problem it solve?
Current all `BlockNodeView` have only position number before the type name, but it actually after node opening tag, this fix show exactly before and after position in the structure view.

``` tsx
// [0]<p>[1]text</p>
<p>text</p>
```
This will a little confuse for new Prosemirror user to identify position,

## What it looks now?
image
image

### CODE REVIEW

File: src/tabs/structure.tsx [hunk 1]
```diff
@@ -65,8 +65,19 @@ const Side = styled("div")({
...
+});
+StartSide.displayName = "Side";
+
+const Bar = styled("div")({
   flexGrow: 1,
+})
+Bar.displayName = "Bar"
+
+const Center = styled("div")({
+//  flexGrow: 1,
```
Comment (d4rkr00t) [inline src/tabs/structure.tsx]: A comment :)

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4005 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (d4rkr00t) [inline yarn.lock]: Repo is not using yarn :)

**Review by d4rkr00t — COMMENTED** [review]
Also prettier failed :)

File: src/tabs/structure.tsx [hunk 3]
```diff
@@ -65,8 +65,18 @@ const Side = styled("div")({
 });
 Side.displayName = "Side";
 
-const Center = styled("div")({
+const StartSide = styled("div")({
```
Comment (d4rkr00t) [inline src/tabs/structure.tsx]: This component is not used anywhere.

**Comment by d4rkr00t** [comment]
Also I caused a bit of conflicts by removing emotion :) I'm sorry...

File: src/tabs/structure.tsx (hunk 3 above)
Comment (mwonng) [reply src/tabs/structure.tsx]: not show it used i think because of merge commits :) should be good now

**Pull request merged**